### ISSUE TITLE
Question: application notification but no application

### ISSUE THREAD

**Issue by Tealk** [LemmyNet/lemmy#3866]
### Question

Hello,

I have received an email notification that there would be a new application, but I do not see any on the lemmy instance.

In the chat someone said that the email comes before the email confirmation link is clicked but the application is displayed only after that is it true?

If so, could you change it so that you only after the confirmation link the email message comes?

**Comment by Nutomic** [comment]
Duplicate of https://github.com/LemmyNet/lemmy/issues/3024
